client/src/components/layout/ReviewsList.js

```diff
@@ -1,10 +1,11 @@
 import React from "react"
 import ReviewTile from "./ReviewTile"
 
-const ReviewsList = ({reviews}) => {
+const ReviewsList = (props) => {
+    const reviews = props.reviews
```

Comment:
why did you change this? the destructuring in the old code is more consistent with React best practices